### DISCUSSION TITLE
transfer the waveform of the oscillator.

### DISCUSSION THREAD

**Discussion by Jpro600k** [image-et-son/p600fw#138] [Q&A] [answered]
I want a tool to switch and transfer the waveform of the oscillator. Is it possible?

**Comment by image-et-son** [comment]
I am not sure what exactly you mean by this. Do you want to modulate the oscillator waveform selection? You can do it be MIDI CC, of course. But maybe you mean something else?

**Reply by Jpro600k** [reply]
> I am not sure what exactly you mean by this. Do you want to modulate the oscillator waveform selection? You can do it be MIDI CC, of course. But maybe you mean something else?

Is it easy to replace and transfer each waveform in both OSC and LFO in the section dealing with basic waveforms with a tool like exchanging samples? Is it possible to realize a tool that reads the OS, replaces the waveform, builds it, and outputs SYX with a simple UI?

**Comment by matrix12x** [comment, marked as answer]
The waveforms of the two oscillator are not code, they are from analog oscillators (CEM3340).

The LFO waveforms are code, however, the LFO waveforms are not tables, (like samples), they are mathematical functions. I don't think this would be a plausible tool to make.
